roles/slurm/files/epilog-exclusive.d/50-docker-containers

```diff
@@ -7,13 +7,16 @@ fi
 
 for id in $(docker ps -aq)
 do
-    if ! docker inspect "$id" --format '{{.HostConfig.CgroupParent}} {{.Name}}' | grep kube
+    if docker inspect "$id" --format '{{.HostConfig.CgroupParent}} {{.Name}}' | grep kube
     then
         continue  # skip kube containers
     fi
 
     logger -s -t slurm-epilog "Removing docker container $id ..."
 
     docker stop "$id" || true
-    docker rm "$id"
+    if docker inspect "$id"
```

Comment:
Only try to remove containers if they still exist, previously this failed if the containers were run with `--rm`